docs/cedarling/quick-start/cedarling-quick-start.md

```
@@ -115,14 +115,14 @@ this.
 
 - Fork the [demo repository](https://github.com/JanssenProject/CedarlingQuickstart). While creating the fork, uncheck the
   `Copy the main branch only` checkbox.
-- Update the `openid_configuration_endpoint` value in the trusted issuer file
+- Update the `configuration_endpoint` value in the trusted issuer file
   [`tarpDemo/trusted-issuers/jans.json`](https://github.com/JanssenProject/CedarlingQuickstart/blob/main/tarpDemo/trusted-issuers/jans.json) in the fork.
   Set it to the `.well-known/openid-configuration` endpoint of your IDP.
 
     For instance:
 
     ```json
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)